var/spack/repos/builtin/packages/nalu-wind/package.py

```diff
@@ -17,7 +17,7 @@ class NaluWind(CMakePackage):
 
     tags = ['ecp', 'ecp-apps']
 
-    version('master', branch='master')
+    version('master', branch='master', submodules=True)
```

Comment:
We don't want to turn on submodules by default.  The mesh repo is around 1GB of data and this will enable and init that submodule for every build. Look at line 149 to see how I got around this with the tests.  You can call the git command directly in another function to just init wind-utils if you want.

Reply:
Sounds good! I changed it to be how you suggest.